spec/models/cart_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'rails_helper'
+
+RSpec.describe Cart, type: :model do
+  it { is_expected.to belong_to(:user) }
+  it { is_expected.to have_many(:order_items) }
+
+  describe "#calculate_total" do
```

Comment:
For a method like this, you should have at least one test with numbers, especially when you have a default of 0 set somewhere (in this case in your factory). It might work beautifully when everything is 0 but not actually add anything up .